packages/pinia/src/types.ts

```diff
@@ -543,34 +543,46 @@ export type _GettersTree<S extends StateTree> = Record<
  */
 export type _ActionsTree = Record<string, _Method>
 
+/**
+ * @internal
+ */
+declare type _ExtractStateFromSetupStore_Keys<SS> = keyof { [K in keyof SS as SS[K] extends _Method | ComputedRef ? never : K]: any }
+
+/**
+ * @internal
+ */
+declare type _ExtractActionsFromSetupStore_Keys<SS> = keyof { [K in keyof SS as SS[K] extends _Method ? K : never]: any }
+
+/**
+ * @internal
+ */
+declare type _ExtractGettersFromSetupStore_Keys<SS> = keyof { [K in keyof SS as SS[K] extends ComputedRef ? K : never]: any }
```

Comment:
shouldn't this and others be a regular types?

Reply:
Don't know what I can't make Prettier working in this project, I just copy the code to other project and do format. 😅

> shouldn't this and others be a regular types?

I think these just attached types of `_ExtractStateFromSetupStore`, `_ExtractActionsFromSetupStore`, `_ExtractGettersFromSetupStore` but not regular types.